server/rooms.ts

```diff
@@ -558,7 +558,7 @@ export abstract class BasicRoom {
 		if (!Users.Auth.isAuthLevel(modjoinSetting)) {
 			Monitor.error(`Invalid modjoin setting in ${this.roomid}: ${modjoinSetting}`);
 		}
-		return Users.globalAuth.atLeast(user, modjoinSetting);
+		return this.auth.atLeast(user, modjoinSetting);
```

Comment:
Wouldn't this cause issues if a subroom were set to modjoin `&`, thus allowing subroom auth to join but not the parent room's owner?

Reply:
Never mind - it looks like groupchats restrict modjoin to +/% so this fix is OK.